test/converters/xcresult3/model3/export.go

```diff
@@ -0,0 +1,35 @@
+package model3
+
+import (
+	"encoding/json"
+	"time"
+)
+
+type TestAttachmentDetails struct {
+	TestIdentifier string       `json:"testIdentifier"`
+	Attachments    []Attachment `json:"attachments"`
+}
+
+type Timestamp time.Time
+
+type Attachment struct {
+	ExportedFileName           string    `json:"exportedFileName"`
+	SuggestedHumanReadableName string    `json:"suggestedHumanReadableName"`
+	IsAssociatedWithFailure    bool      `json:"isAssociatedWithFailure"`
+	Timestamp                  Timestamp `json:"timestamp"`
+	ConfigurationName          string    `json:"configurationName"`
+	DeviceName                 string    `json:"deviceName"`
+	DeviceID                   string    `json:"deviceId"`
+	RepetitionNumber           int       `json:"repetitionNumber"`
+}
+
+func (t *Timestamp) UnmarshalJSON(b []byte) error {
+	var timestamp float64
+	if err := json.Unmarshal(b, &timestamp); err != nil {
+		return err
+	}
+
+	*t = Timestamp(time.Unix(int64(timestamp), 0))
+
+	return nil
+}
```

Comment:
I wrote above that for the test cases we receive human readable string for duration. Attachments have a proper unix timestamp. We do not use the timestamp at the moment but thought that I will implement everything here as well.

Go cannot parse a unix timstamp by default and requires manual unmarshalling.